data/strings/strings.txt

```diff
@@ -1846,6 +1846,47 @@
     zh-Hans = 书签
     zh-Hant = 書籤
 
+  [bookmarks_and_tracks]
+    comment = "Bookmarks" dialog title, please also edit it in iphone/plist.txt
```

Comment:
The same comment for [bookmarks] should be changed to reflect the actual situation.

Reply:
up